src/platforms/python/common/configuration/integrations/loguru.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Loguru
+description: Learn about using Sentry with loguru.
+sidebar_order: 80
+---
+
+The [Loguru](https://github.com/Delgan/loguru#readme) integration lets you capture log messages and send them to Sentry.
+
+Most of the work is handled by the [`logging`](/platforms/python/guides/logging) integration, so most of the examples there will work with loguru.
```

Comment:
```suggestion
The Python [Logging](/platforms/python/guides/logging) integration provides most of the Loguru functionality and most examples on that page work with Loguru.
```

Reply:
While correction `loguru -> Loguru` is right, the `logging -> Logging` is not. Because official Loguru docs mention its name with first capital character and `logging` docs mention itself as just logging (without capital and \`). I think it's quite confusing to use the word "logging" and library `logging` in one style, so isn't it better to mention `logging` with `?